go/test/endtoend/backup/vtbackup/backup_only_test.go

```diff
@@ -258,7 +272,7 @@ func restore(t *testing.T, tablet *cluster.Vttablet, tabletType string, waitForS
 	log.Infof("restoring tablet %s", time.Now())
 	resetTabletDirectory(t, *tablet, true)
 
-	err := tablet.VttabletProcess.CreateDB(keyspaceName)
+	err := tablet.VttabletProcess.CreateDBWithSuperReadOnly(keyspaceName)
```

Comment:
Another place where it seems like we're *avoiding* testing the new behavior, no?

Reply:
fixed